code/modules/surgery/surgery.dm

```diff
@@ -93,14 +93,14 @@
 	if(type in opcomputer.advanced_surgeries)
 		return TRUE
 
-/datum/surgery/proc/next_step(mob/user, intent)
+/datum/surgery/proc/next_step(mob/living/user, disarm)
 	if(location != user.zone_selected)
 		return FALSE
 	if(step_in_progress)
 		return TRUE
 
 	var/try_to_fail = FALSE
-	if(intent == INTENT_DISARM)
+	if(user.in_combat_mode()) //This means you are fucking up on purpose (you combat mode right clicked)
```

Comment:
Is this meant to be checking the `disarm` parameter?

Reply:
Oh, no i was supposed  to remove that param, I realized there's no way next_step() in combat mode unless it was a disarm, so I don't need to pass that info.